app/components/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```

Comment:
In the tutorial I followed, the guy claimed it as a convention to capitalise the name of the file  if its a component.